### PR TITLE
Add filecoin-project/lassie

### PR DESCRIPTION
**Pull request by rvagg**

### CODE REVIEW

**Review by hannahhoward — APPROVED** [review]

**Review by kylehuntsman — APPROVED** [review]

**Comment by galargh** [comment]
Before we merge, could you confirm `web3-bot` already has `push` access to `filecoin-project/lassie`?

**Comment by hannahhoward** [comment]
I added @web3-bot to have write access.

**Pull request merged**
